Doppler.Push.Api/appsettings.json

```
@@ -16,5 +16,6 @@
     "PublicKeysFolder": "public-keys",
     "PublicKeysFilenameRegex": "\\.xml$"
   },
-  "FirebaseCredencialAdminSdk": "firebase-secret-keys/firebase-adminsdk.json"
+  "FirebaseCredential": {
+  }
```

What do you think about adding here the example value?

```suggestion
  "FirebaseCredential": {
    "type": "service_account",
    "project_id": "REPLACE_WITH_FIREBASE_PROJECT_ID",
    "private_key_id": "REPLACE_WITH_PRIVATE_KEY_ID",
    "private_key": "REPLACE_WITH_PRIVATE_KEY",
    "client_email": "REPLACE_WITH_ADMINSDK_ALIAS_CLIENT_EMAIL@REPLACE_WITH_ADMINSDK_DOMAIN_SERVICE_ACCOUNT_CLIENT_EMAIL",
    "client_id": "REPLACE_WITH_CLIENT_ID",
    "auth_uri": "https://accounts.google.com/o/oauth2/auth",
    "token_uri": "https://oauth2.googleapis.com/token",
    "auth_provider_x509_cert_url": "https://www.googleapis.com/oauth2/v1/certs",
    "client_x509_cert_url": "https://www.googleapis.com/robot/v1/metadata/x509/REPLACE_WITH_ADMINSDK_ALIAS_CLIENT_EMAIL%40REPLACE_WITH_ADMINSDK_DOMAIN_SERVICE_ACCOUNT_CLIENT_EMAIL"
}
```

---

I included all values except `private_key`

---

What do you think about adding it as we do with other secrets?

---

I decided no modify these key/values in this PR